### DISCUSSION TITLE
Create FargateService with existing FargateTaskDefinitionArn

### DISCUSSION THREAD

**Discussion by pahud** [aws-samples/aws-cdk-examples#651] [Q&A]
## :question: Guidance Question

I am trying to create a `ecs.FargateService` with existing `fargateTaskDefinitionArn`. However, the `taskDefinition` property expects `ecs.TaskDefintion` but `fromFargateTaskDefinitionArn()` returns `ecs.IFargateTaskDefinition`.

Any sample or guidance is appreciated.

<img width="818" alt="圖片" src="https://user-images.githubusercontent.com/278432/64251029-23ffb180-cf4a-11e9-93c8-322ced890a55.png">

<img width="910" alt="圖片" src="https://user-images.githubusercontent.com/278432/64251537-5cec5600-cf4b-11e9-80aa-19c3259edeec.png">

### Environment

  - **CDK CLI Version:** 1.6.1 (build a09203a)
  - **Example:** 
  - **Example Version:** 
  - **OS:** OSX Mojave
  - **Language:** TypeScript

**Comment by rragundez** [comment]
Any update on fixing this bug?

**Comment by pahud** [comment]
Let's watch and comment on this issue
https://github.com/aws/aws-cdk/issues/6240
